raspberry/bzzz/sensors/time_of_flight_sensor.py

```diff
@@ -0,0 +1,196 @@
+from bzzz.sensors.filters import median_filter
+from time import time_ns, sleep
+from datetime import datetime
+import VL53L0X
+import os
+
+
+class TimeOfFlightSensor:
+    # NOTE: This class fetches altitude in millimeters and this class requires i2c interfacing with the sensor.
+    def __init__(self,
+                 num_latest_readings_to_keep: int = 3,
```

Comment:
wouldn't `median_filter_length` be a better name?

Reply:
Done!